manifests/4.10/kube-descheduler-operator.crd.yaml

```diff
@@ -47,6 +47,13 @@ spec:
                   description: managementState indicates whether and how the operator should manage the component
                   type: string
                   pattern: ^(Managed|Unmanaged|Force|Removed)$
+                mode:
+                  description: Modes configures the descheduler to either evict pods or to simulate the eviction
+                  type: string
+                  enum:
```

Comment:
We probably want `Predictive` to be the default, all the rest lgtm

Reply:
Done